cmd/lytfs/main.go

```diff
@@ -31,10 +34,15 @@ func main() {
 	ctx, cancel := context.WithTimeout(context.Background(), *timeout)
 	defer cancel()
 
-	fcp := yagfuncdata.DefaultFileContentProvider
+	owner := strings.Split(*repo, "/")[0]
+	repo := strings.Split(*repo, "/")[1]
+
+	fcp := yagfuncdata.NewGitHubFileProvider(github.NewClient(nil), owner, repo, *branch)
```

Comment:
Not sure if this is the best way to go about it, but at this point it seems quite silly to call `DefaultFileContentProvider` if we have the defaults set via flags introduced in this change.

Reply:
Yes, I'm fine to leave `yagfuncdata.DefaultFileContentProvider` out.